src/plugins/interaction/GraphInteraction/CMakeLists.txt

```diff
@@ -12,7 +12,8 @@ add_library(${LIBRARY_NAME} SHARED
   ${SRCS}
 )
 
-include_directories(${Boost_INCLUDE_DIRS})
+target_include_directories(scrimmage-boost
+  INTERFACE ${Boost_INCLUDE_DIRS})
```

Comment:
This library needs boost so `INTERFACE` should be `PUBLIC`

Reply:
I think I finally got it. Still a bit confused about target_include_directories and the many many things that can go into them (e.g. all of https://cmake.org/cmake/help/v3.0/command/target_include_directories.html), but that was a good learning experience. Thank you!